<launch>
  <arg name="box_namespace" default="gt_box"/>
  <group ns="$(arg box_namespace)">
    <node pkg="box_health" type="publish_local_health_status.py" name="box_health_npc" output="screen">
      <param name="capability_group" value="Monitoring"/>
    </node>
  </group>
</launch>